lib/charms/tls_certificates_interface/v4/tls_certificates.py

```diff
@@ -1763,6 +1828,7 @@ def __init__(self, charm: CharmBase, relationship_name: str):
         self.framework.observe(charm.on.update_status, self._configure)
         self.charm = charm
         self.relationship_name = relationship_name
+        self._security_logger = _OWASPLogger(application=charm.app.name)
```

Comment:
I'm just wondering if it's a little weird that we're logging events under the charm's scope. If you saw your charm producing these events, but you didn't write the code, would you be confused?

Maybe this makes sense, it's just something that came to mind as perhaps a little odd.